featuretools/primitives/base/utils.py

```diff
@@ -1,25 +1,22 @@
 try:
-    # python 3.7 deprecated getargspec
-    from inspect import getfullargspec as getargspec
+    # python 3
+    from inspect import signature
 except ImportError:
-    # python 2.7 - 3.6 backwards compatibility import
-    from inspect import getargspec
+    # python 2
+    from funcsigs import signature
```

Comment:
We should add this module as a dependency for python2

Reply:
Okay sounds good.

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/481/commits/fa7d29d44f24e6d298332aaa9fc385262c590e0f#diff-b4ef698db8ca845e5845c4618278f29aR13)